website/templates/peak_view.html

```diff
@@ -115,7 +115,13 @@ <h6 class="p-3 mb-0">Messages</h6>
                     <div><a href=coach_dashboard.html>Soccer (M)</a></div>
                     <div><a href="coach_dashboard.html">Soccer(W)</a></div>
                     <div><a href="coach_dashboard.html">Track and Field</a></div>
-                    <div><a href="coach_dashboard.html">Cross Country</a></div>
+                    <div><a href="coach_dashboard.html">Cross Country</a></div> -->
+
+                    {% for team in teams %}
+                      <div><a href="coach_dashboard/{{ team.id }}">{{ team.name }}</a></div>
```

Comment:
This is good, but we need to change things in views.py for it to work

Reply:
cool